### ISSUE TITLE
Question: does it support other utf-8 natual language? 

### ISSUE THREAD

**Issue by yingshaoxo** [karpathy/minGPT#115]
For example, chinese or japanese?

**Comment by tyeestudio** [comment]
yes, it does support both chinese or japanese. I am currently running test for tokens which contains chinese , japanese and english. training works, prediction also works.
